### DISCUSSION TITLE
Expose raw values for Font Scale and Pixel Ratio

### DISCUSSION THREAD

**Discussion by levibuzolic** [jpudysz/react-native-unistyles#187] [Feature Requests]
👋  Jacek.

I was wondering if you would consider exposing the raw values for pixel ratio/density and font scale?

The recently added `contentSizeCategory` is very useful, but I was a little surprised to see the underling pixel ratio and font scale values not being exposed along side it.

I'm currently using PixelRatio and FontScale to manually scale some UI elements while having greater control over min/max scales.

I currently solve this in user-land with a hook that reads `PixelRatio.get()` and `PixelRatio.getFontScale()` anytime the application is foregrounded. I could read these values in a dynamic stylesheet call, but if the scales don't change enough to fall into a new content size category, I won't get updated values to use.

Having these exposed on `UnistylesRuntime` would mean this logic could be moved into the stylesheet (where I think it'd more naturally belong).

**Comment by jpudysz** [comment]
Hey 👋

I'm considering doing this in the near future. I want to expose more and more features to `UnistylesRuntime`. I'll get started on that soon.

**Reply by jpudysz** [reply]
Implemented in the latest RC
